Vagrantfile

```diff
@@ -36,7 +36,7 @@ Vagrant.configure('2') do |config|
   # For a complete reference, please see the online documentation at
   # https://docs.vagrantup.com.
 
-  config.vm.provision 'ansible_local', run: 'always' do |ansible|
+  config.vm.provision 'main', type: 'ansible', run: 'always' do |ansible|
```

Comment:
It is to make it known that this is the main script, as there is another update script below. Not deemed necessary, but just to avoid confusion.